lib/grovepi/dht_11/default_trigger.ex

```diff
@@ -0,0 +1,39 @@
+defmodule GrovePi.DHT11.DefaultTrigger do
```

Comment:
I'd be a little worried about flutter, but I haven't been able to test this on hardware. I'm not sure how often the temp and humidity change, but maybe if it is right on the edge of a level I could see a bunch of messages. Maybe we should make the default only trigger if the number holds for a certain number of checks? I could be out of my mind.

Could we make this send multiple events? `:temp_changed`, `:humidity_changed`, `:both_changed`?